pkg/ccl/backupccl/restore_span_covering_test.go

```diff
@@ -225,6 +225,9 @@ func checkRestoreCovering(
 	}
 	var spanIdx int
 	for _, c := range cov {
+		if len(c.Files) > 500 {
```

Comment:
We could move the 200 limit to a constant and test explicitly against that.

Reply:
200 is a target, above which we won't _choose_ to add extend the RHS of a cover span to the next interesting point, but it isn't a hard limit is it? if there are >200 files that all overlap the span just between two "interesting points" then we still need to make a >200 file span.

500 is small enough that it easily fails this test before the change, and passes after, without needing to think much about if it is 201 or 203 vs 199.